lib/Mojo/WebSocketProxy/Dispatcher/Config.pm

```diff
@@ -0,0 +1,59 @@
+package Mojo::WebSocketProxy::Dispatcher::Config;
+
+use Mojo::Base -base;
+
+my $actions;
+my $config;
+
```

Comment:
O, man! Don't use the global config, please. Store config and actions inside your Config object instance